spec/acceptance/device_spec.rb

```diff
@@ -65,8 +65,10 @@
 
       it 'outputs resources as yaml' do
         stdout_str, status = Open3.capture2e("puppet resource #{common_args} device_provider --to_yaml")
-        expected_values = "device_provider: |2\n\s+wibble:\n\s+ensure: :present\n\s+string: sample\n\s+string_ro: fixed\n\s+string_param: default value\n"
-        expect(stdout_str.strip).to match %r{#{expected_values}\Z}
+        expected_values = 'device_provider: |2\n\s+wibble:\n\s+ensure: :present\n\s+string: sample\n\s+string_ro: fixed\n\s+string_param: default value'
```

Comment:
@DavidS It seems the YAML generation on windows produces a slightly different output.

1. Linux / MacOS: 
```yaml
device_provider: |2
  wibble:
  ensure: :present
   string: sample
   string_ro: fixed
```

1. Windows: 
```yaml
device_provider:
  wibble:
  ensure: :present
   string: sample
   string_ro: fixed
```

I've tinkered with Rubular but didn't manage to come up with a regexp covering both cases. Any suggestions welcome!